lib/control/validation/fetch.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const HTTP_BAD_REQUEST = 400;
```

Comment:
you're right. I should probably extract all the BAD_REQUEST responses into a separate module like with the method not allowed module.